### ISSUE TITLE
TypeScript Getters and Setters do not render correctly with TypeDoc

### ISSUE THREAD

**Issue by ltfschoen** [TypeStrong/typedoc#882]
In our [codebase](https://github.com/polkadot-js/api), we are using TypeDoc to generate a Gitbook. In the development environment, using that codebase we can preview the Gitbook generated by TypeDoc by installing relevant dependencies, building and serving the Gitbook with `yarn; yarn run build; yarn gitbook serve` and then viewing the Gitbook at http://localhost:4000. The production version of the GitBook is at published at https://polkadot.js.org/api/. 

We encountered an issue, whereby if we apply TypeDoc to TypeScript Getters `get` and Setters `set`, TypeDoc doesn't appear to correctly recognise them and so does not render them correctly in the generated Gitbook pages.

For example, in the following code snippet we define a Getter with `get provider ()` that returns the value of `_provider`:
```
export default class Rpc implements RpcInterface {
  private _provider: ProviderInterface;

  constructor (provider: ProviderInterface) {
    this._provider = provider;
  }

  /**
   * @example
   * <BR>
   *
   * ```javascript
   * <insert_example_here>
   * ```
   */
  get provider (): ProviderInterface { return this._provider; }
...
``` 
However in Gitbook page it, whilst it correctly lists it under the heading "Accessors", it renders it as `getprovider()` instead 
of `get provider()` (i.e. there's no space between, so it doesn't appear to be recognising the Getter correctly). See screenshot below:

<img width="299" alt="screen shot 2018-10-10 at 12 04 40" src="https://user-images.githubusercontent.com/6226175/46729873-d79ee980-cc86-11e8-81e0-b356efcafa73.png">

So this is a problem because a user reading the the Gitbook documentation may think the method name is `getprovider()` when it's actually just `provider()`

We are using:
* TypeScript 3.0.1
* TypeDoc 0.12.0

**Comment by ltfschoen** [comment]
I tried looking into this issue myself. I looked through the Typedoc codebase to find where getters and setters are processed, and then added console.log's in the relevant parts of the JavaScript files of the node_modules/typedoc dependency in our codebase before building the Gitbook.

I found where it generates the method name of the [getter on Line 172 of lib/models/reflections/declaration.ts](https://github.com/TypeStrong/typedoc/blob/master/src/lib/models/reflections/declaration.ts#L172) and [setter on Line 176 of lib/models/reflections/declaration.ts](https://github.com/TypeStrong/typedoc/blob/master/src/lib/models/reflections/declaration.ts#L176), and tried prepending a blank space in front of it. But it didn't render the blank space so somewhere else in the codebase trims the whitespace from the method name.
I tried using `\u00A0` instead of just ` ` but it still didn't render. But when I prepended a letter accompanied before the space `P ` it rendered and I could see what was being affected. See the rendered output and associated code changes below:
<img width="309" alt="screen shot 2018-10-14 at 01 43 54" src="https://user-images.githubusercontent.com/6226175/46911227-76606a00-cf55-11e8-80bb-f41bff694f19.png">
<img width="693" alt="screen shot 2018-10-14 at 01 47 16" src="https://user-images.githubusercontent.com/6226175/46911230-8710e000-cf55-11e8-9c84-f3e0d03fb585.png">

It would be more ideal to just add a blank space after the `get ` and `set `, since prepending the blank space before the method name instead causes the blank space to appear with the method name everywhere else that the method name is rendered (as shown in the screenshot).
